### ISSUE TITLE
New resource is decorated when form is reloaded

### ISSUE THREAD

**Issue by alessandro1997** [activeadmin/activeadmin#4307]
When a new resource has validation errors and the page is reloaded, the resource gets decorated and the decorated values appear in the fields in place of the user's original input.

This can be fixed by undecorating the resource at the beginning of the ```form``` block:

```ruby
form do |f|
  f.object = f.object.object if f.object.respond_to?(:object)

  # ...
end
```

Still, this shouldn't be happening. Is this a known bug? I've done some research but didn't find anything.

**Comment by seanlinsley** [comment]
As of #2085 you can pass `decorate: false` to the form to do the same thing.

**Comment by seanlinsley** [comment]
Actually #2905 and its PR, #2911, stopped decorating forms, which was then followed up with #2912.

What version of Active Admin are you using?

**Comment by alessandro1997** [comment]
```console
$ grep activeadmin Gemfile.lock 
remote: git://github.com/activeadmin/activeadmin.git
  activeadmin (1.0.0.pre2)
activeadmin!
```

**Comment by seanlinsley** [comment]
This should be fixed by #4287. Please let me know if it's still happening on the latest master.
